integration-tests/src/test/scala/eu/ostrzyciel/jelly/integration_tests/util/TestComparable.scala

```diff
@@ -0,0 +1,8 @@
+package eu.ostrzyciel.jelly.integration_tests.util
+
+/**
+ * A trait for comparing two objects of the same type in the context of a scalatest test.
+ * @tparam T the type of the objects to compare
+ */
+trait TestComparable[T] extends Measure[T]:
```

Comment:
does it make sense for these two small traits, one extending the other, to be in separate files?

Reply:
eh, let's keep it for now. Can refactor it later, it's just test code.